examples/custom-state-storage/mongo-state-storage.js

```diff
@@ -0,0 +1,62 @@
+const mongodb = require('mongodb');
+const MongoClient = mongodb.MongoClient;
+const Bluebird = require('bluebird');
+
+Bluebird.promisifyAll(MongoClient);
```

Comment:
Bluebird is not necessary here, and IMO makes the example more complicated than it needs to be.  If the `mongodb` package does not expose a promise interface (which I would be surprised at), then I would just use callbacks.  If you really want to use promises, you could also use `util.promisify`, since that is a built-in package.

Reply:
Also, if you want to use promises, I would prefer the example to be `async`/`await` than raw promises.

Reply:
used async await instead of BludBird.